plugins/node.d/amavis

```diff
@@ -31,36 +33,34 @@ plugin usually runs at.  To enable log reading it needs to run as a
 group or user that has read access, as shown above.
 
 By default the logtail program is started without any explicit path,
-but if it is not found in the system $PATH then you will need to specify
+but if it is not found in the system $PATH then you'll need to specify
```

Comment:
Back then I was taught in school, that this abbreviation is mainly suitable in spoken communication and not for written text :)
But you could convince me, if you are a native speaker and/or very sure, about what you are doing :)